docs/analyzer/index.md

```diff
@@ -2,27 +2,159 @@
+- [Getting started](#getting-started)
+  - [Running in Python](#running-in-python)
+  - [Running Presidio as an HTTP server](#running-presidio-as-an-http-server)
+    - [Using docker container](#using-docker-container)
+    - [Using python runtime](#using-python-runtime)
+- [Creating PII recognizers](#creating-pii-recognizers)
+- [Outputting the analyzer decision process](#outputting-the-analyzer-decision-process)
+- [Supported entities](#supported-entities)
+- [API reference](#api-reference)
+
 ## Installation
 
-## Getting started
+### Using pip
+
+> Consider installing the Presidio python packages on a virtual environment like venv or conda.
+
+To get started with Presidio-analyzer,
+download the package and the `en_core_web_lg` spaCy model:
+
+```sh
+pip install presidio-analyzer
+python -m spacy download en_core_web_lg
+```
+
+### Using Docker
+
+> This requires Docker to be installed. [Download Docker](https://docs.docker.com/get-docker/).
+
+```sh
+# Download image from Dockerhub
+docker pull mcr.microsoft.com/presidio-analyzer
 
-- Running in Python.
-- Running using Docker.
-- Creating PII recognizers.
-- [Supporting additional languages and ML models](languages.md)
-- [Outputing the analyzer decision process](decision_process.md)
-- API reference.
+# Run the container with the default port
+docker run -d -p 5001:5001 mcr.microsoft.com/presidio-analyzer:latest
+```
+
+### From source
+
+First, clone the Presidio repo. [See here for instructions](../installation.md#install-from-source).
+
+Then, build the presidio-analyzer container:
+
+```sh
+cd presidio-analyzer
+docker build . -t presidio/presidio-analyzer
+```
+
+## Getting started
 
 ### Running in Python
 
-### Running using Docker
+Once the Presidio-analyzer package is installed, run this simple analysis script:
+
+```python
+from presidio_analyzer import AnalyzerEngine
+
+# Set up the engine, loads the NLP module (spaCy model by default) and other PII recognizers
+analyzer = AnalyzerEngine()
+
+# Call analyzer to get results
+results = analyzer.analyze(text="My phone number is 212-555-5555",
+                           entities=["PHONE_NUMBER"],
+                           language='en')
+print(results)
+
+```
+
+### Running Presidio as an HTTP server
+
+You can run presidio analyzer as an http server using either python runtime or using a docker container.
+
+#### Using docker container
+
+```sh
+cd presidio-analyzer
+docker run -p 5001:5001 presidio-analyzer 
+```
+
+#### Using python runtime
+
+> This requires the Presidio Github repository to be cloned.
+
+```sh
+cd presidio-analyzer
+python app.py
+curl -d '{"text":"John Smith drivers license is AC432223", "language":"en"}' -H "Content-Type: application/json" -X POST http://localhost:3000/analyze
+```
+
+## Creating PII recognizers
+
+- For a tutorial on adding new PII recognizers, [click here](adding_recognizers.md).
+- Fpr best practices for developing new recognizers, [click here](developing_recognizers.md).
```

Comment:
Maybe remove the *For in each sentence
A tutorial on adding...
Best practices for ...
Multi-language support ....

Reply:
Agree, added it as suggestions, also suggest removing the click here - the whole sentence can be a link